iroh-net/src/hp/magicsock/conn.rs

```diff
@@ -2826,7 +2820,7 @@ mod tests {
         })
     }
 
-    fn setup_logging() {
+    pub fn setup_logging() {
```

Comment:
I kind of wish you copied in `iroh-bytes/src/test_utils.rs` to `iroh-net/src/test_utils.rs` and used it's `setup_logging` call for your functions instead of making this pub...  But I tried to clean this up earlier and utterly failed failed.  For new `#[tokio::test]` tests it should just work though, since they're single-threaded runtimes.  Just don't try change existing tests.

But if you don't want to do that now that's fine, it's my fault for not managing to clean this up before.

Reply:
Yes this sounds good to me too. I also want to expose `run_derp_and_stun` in a test_util module also to other crates (i.e. iroh-gossip) that use iroh-net and need to test things with a local derper. 
However I'd prefer to do this after the merge in a new PR.